charts/prometheus/templates/prometheus-networkpolicy.yaml

```
@@ -42,7 +42,7 @@ spec:
     - podSelector:
         matchLabels:
           tier: nginx
-          component: ingress-controller
+          component: {{ ternary "cp-ingress-controller" "dp-ingress-controller" (or (eq .Values.global.plane.mode "unified") (eq .Values.global.plane.mode "control") )}}
```

i know current code is fine, but for me this is simpler :)
```suggestion
          component: {{ ternary "dp-ingress-controller" "cp-ingress-controller" (eq .Values.global.plane.mode "data") )}}
```